### PR TITLE
update to dotnet 6

### PR DESCRIPTION
**Pull request by jhgbrt**
updated target framework in all projects
simplified Program.cs and removed Startup.cs 
(as per net5 -> net6 migration guidance)

### CODE REVIEW

**Comment by xperiandri** [comment]
I don't think that it is a good idea to remove `Startup.cs`
The simplified template for dummies in .NET 6 is not what a professional Orchard Core project requires.

**Comment by jhgbrt** [comment]
Well, it's a sample intended to demonstrate how OrchardCore Commerce can be used. I would think that especially new developers would expect it to align with the simplified approach (which for better or worse, newcomers will be familiar with in the future).

<opinion>
I believe the bootstrap magic dance between Program and Startup added little to no value. I like the way you now have one place where you can find all bootstrap logic in the new template & philosophy, even for 'advanced' projects.
</opinion>

Of course, I'll leave it to the project maintainers to apply this PR or not.

File: SampleWebApp/Program.cs
Comment (Skrypt) [inline SampleWebApp/Program.cs:17]: else if (env.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

File: SampleWebApp/Program.cs
Comment (Skrypt) [inline SampleWebApp/Program.cs:24]: new line at eof

**Comment by Skrypt** [comment]
This will probably break if we multi-target frameworks. I think it is better to keep the startup.cs file.

**Review by Skrypt — APPROVED** [review]

**Comment by jhgbrt** [comment]
I don't understand your concern. What does the sample web app have to do with multi-targeting?

**Comment by Skrypt** [comment]
We want to be able to test it with the CI by building it with netcoreapp3.1, net5.0, net6.0. And these changes will only work in net6.0 as far as I know?

**Comment by agriffard** [comment]
See https://github.com/OrchardCMS/OrchardCore/issues/10761 for the same concern on the main repo.

**Comment by Skrypt** [comment]
All is good here too. We can merge. I suggest we keep a discussion open about the new way of bootstrapping the app.
Next PR, multi-target.

**Pull request merged**
